submissions/onyemeMichael/domManipulation/product.js

```diff
@@ -58,6 +58,5 @@ function fixIndex() {
     priceField();
     shippingAmount();
     totalCost();
-}
```

Comment:
This function doesn't have a closing `}`

Reply:
The closing parenthesis is there @adaugochi